src/jobs/BenchmarkJob.jl

```diff
@@ -213,7 +213,7 @@ function Base.run(job::BenchmarkJob)
                 end
                 !(found_previous_date) && error("didn't find previous daily build data in the past 31 days")
             catch err
-                rethrow(NanosoldierError("encountered error when retrieving old daily build data", err))
+                nodelog(cfg, node, "encountered error when retrieving old daily build data: " * sprint(showerror, err))
```

Comment:
Actually, maybe we should keep the error + `rethrow` here (since we still want to contextualize *actual* errors that occur in the try block. Instead, we should change the `error` on line 214 to a `nodelog`.

Reply:
Good call. I've made that change now.